src/AppHarness/StoreProvider.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { PropsWithChildren, useEffect, useState } from "react";
+import {
+  useGetAccountInfo,
+  useGetLoginInfo,
+  useGetNetworkConfig,
+  useGetPendingTransactions,
+} from "@multiversx/sdk-dapp/hooks";
+import { contractsForChain, convertWeiToEsdt } from "libs/util";
+import { getAccountTokenFromApi } from "MultiversX/api";
+import { DataNftMarketContract } from "MultiversX/dataNftMarket";
+import { useAccountStore, useMarketStore, useMintStore } from "store";
+import { useChainMeta } from "store/ChainMetaContext";
+import { useUser } from "store/UserContext";
+import { DataNftMintContract } from "MultiversX/dataNftMint";
+
+export const StoreProvider = ({ children }: PropsWithChildren) => {
+  const { chainID } = useGetNetworkConfig();
+  const { address } = useGetAccountInfo();
+  const { hasPendingTransactions } = useGetPendingTransactions();
+  const { chainMeta } = useChainMeta();
```

Comment:
Won't this be called whenever a new page is rendered? Shouldn't these be saved in the store itself, too at login?

Reply:
Okay